portfolio/src/main/webapp/script.js

```diff
@@ -33,11 +33,39 @@ function addRandomFunFact() {
   funFactContainer.innerText = funFact;
 }
 
+/** Accordion / collapsible template from https://www.w3schools.com/howto/howto_js_accordion.asp */
+
+// jQuery - Initialize accordion / collapsible
+if($('body').is('.blog')) {
+  showAccordion();
+}
+
+/** Display accordion / collapsible */
+function showAccordion() {
+  var accordion = document.getElementsByClassName("accordion");
+
+  // Open chosen panel(s) and hide all other panels
+  for (var accordionIndex = 0; accordionIndex < accordion.length; accordionIndex++) {
+    accordion[accordionIndex].addEventListener("click", function() {
+      this.classList.toggle("panel-active");
+      var panel = this.nextElementSibling;
+      if (panel.style.maxHeight) {
+        panel.style.maxHeight = null;
+      }
+      else {
+        panel.style.maxHeight = panel.scrollHeight + "px";
+      }
```

Comment:
optional: if you'd like, you can use the ternary (also called conditional) operator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

ex: panel.style.maxHeight = panel.style.maxHeight ? null : panel.scrollHeight + "px";

It's optional though because even though it's shorter, it's not always more readable (in this case, I think they're both equally readable because it's a simple condition). But up to your personal preference. Just thought I'd point it out in case you weren't aware. This also exists in most other programming languages.

Reply:
Thank you for pointing this out! I remember seeing this in one of my intro courses but didn't use it too much. However, it only takes up 1 line of code compared to the current 6 lines of code so I will go ahead and make that change